### ISSUE TITLE
WebPlat charter: Liaise with AG WG

### ISSUE THREAD

**Issue by LJWatson** [w3c/charter-html#148]
Feedback from the AC review:
The Accessibility Guidelines Working Group (AG WG) should be included in the Liaisons section.

**Comment by chaals** [comment]
I'm not sure why that group should specifically be a listed liaison. @michael-n-cooper, @awkawk, @joshueoconnor, thoughts?

**Comment by joshueoconnor** [comment]
Thanks @challs. I guess it's good for both groups to be aware of areas of mutual interest and/or impact. This happens informally and in a rather ad hoc manner so I'm not averse to being more systematic. 

As a requirement, i don't see a burning imperative but I guess more interaction/communication would be useful IMO. How this happens, via liason etc doesn't bother me.

**Comment by chaals** [comment]
It is a good idea for the Web Platform people to talk to all kinds of people. However, unless there is a specific reason for them to coordinate on something, I prefer not to make it a charter requirement. Otherwise the charter gets filled with stuff that's a good idea and it is harder to work out what the group actually does.

I suggest we close this issue with no action.

**Comment by awkawk** [comment]
Why then do you need to liaise with the Web Security Interest Group, the Web Application Security Working Group, and others? I would think that a simple "This group provides advice or review to ensure that specifications meet the needs of an accessible Web" similar to what is provided for the internationalization WG would make sense.

**Comment by chaals** [comment]
Hi @awkawk,

> Why then do you need to liaise with the Web Security Interest Group, the Web Application Security Working Group, and others

Because we explicitly ask them to do review on our specs, (and expect them to do it). As I understand it, for accessibility that role is filled by the APA group, and we leave the other groups to get on with their work.

**Comment by awkawk** [comment]
Fair enough, so long as that happens I'm ok with that.

**Comment by LJWatson** [comment]
@awkawk, The charter includes APA WG as the formal liaison for accessibility spec reviews, but to the best of my knowledge, AG WG doesn't do spec reviews? If it's something AG WG would be interested in doing, I'd suggest handling it informally (so no undue expectation or burden is placed on participants of the AG WG), but if you feel strongly that AG WG should be providing formal review of WebPlat specs then I don't see any problem with formalising that in the charter.

**Comment by awkawk** [comment]
Yes, I'm just sticking my foot in my mouth. APA is the right group to review. I mostly want to make sure that there is a conduit for communication about technology needs that cut across the user agent / content boundary, but that can be through github issues or other communication and doesn't need to be formalized since it isn't regular enough for the overhead of an established connection. Carry on... :)

**Comment by joshueoconnor** [comment]
Me 2. @chaals has a point about APA doing cross working group a11y review. Fine with me to close this also, as long as that happens. 
Josh -  Sent from a mobile device and I'm all thumbs
-------- Original message --------From: Andrew Kirkpatrick <notifications@github.com> Date: 12/07/2017  22:12  (GMT+00:00) To: w3c/charter-html <charter-html@noreply.github.com> Cc: joshueoconnor <joshue@fastmail.fm>, Mention <mention@noreply.github.com> Subject: Re: [w3c/charter-html] WebPlat charter: Liaise with AG WG (#148) 
Fair enough, so long as that happens I'm ok with that.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/w3c/charter-html","title":"w3c/charter-html","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/w3c/charter-html"}},"updates":{"snippets":[{"icon":"PERSON","message":"@awkawk in #148: Fair enough, so long as that happens I'm ok with that."}],"action":{"name":"View Issue","url":"https://github.com/w3c/charter-html/issues/148#issuecomment-314898530"}}}
